edx_django_utils/monitoring/code_owner/tests/test_middleware.py

```diff
@@ -195,46 +195,33 @@ def test_code_owner_no_mappings(self, mock_set_custom_attribute):
     )
     @patch('edx_django_utils.monitoring.code_owner.middleware.set_custom_attribute')
     def test_no_resolver_for_path_and_no_transaction(self, mock_set_custom_attribute):
-        with patch(
-                'edx_django_utils.monitoring.code_owner.utils._PATH_TO_CODE_OWNER_MAPPINGS',
-                _process_code_owner_mappings()
-        ):
-            request = RequestFactory().get('/bad/path/')
-            self.middleware(request)
-            self._assert_code_owner_custom_attributes(
-                mock_set_custom_attribute, has_path_error=True, has_transaction_error=True
-            )
+        request = RequestFactory().get('/bad/path/')
+        self.middleware(request)
+        self._assert_code_owner_custom_attributes(
+            mock_set_custom_attribute, has_path_error=True, has_transaction_error=True
+        )
 
     @override_settings(
         CODE_OWNER_MAPPINGS={'team-red': ['*']},
     )
     @patch('edx_django_utils.monitoring.code_owner.middleware.set_custom_attribute')
     def test_catch_all_instead_of_errors(self, mock_set_custom_attribute):
-        with patch(
-                'edx_django_utils.monitoring.code_owner.utils._PATH_TO_CODE_OWNER_MAPPINGS',
-                _process_code_owner_mappings()
-        ):
-            request = RequestFactory().get('/bad/path/')
-            self.middleware(request)
-            self._assert_code_owner_custom_attributes(mock_set_custom_attribute, expected_code_owner='team-red')
+        request = RequestFactory().get('/bad/path/')
+        self.middleware(request)
+        self._assert_code_owner_custom_attributes(mock_set_custom_attribute, expected_code_owner='team-red')
 
     @override_settings(
         CODE_OWNER_MAPPINGS=['invalid_setting_as_list'],
         ROOT_URLCONF=__name__,
     )
     @patch('edx_django_utils.monitoring.code_owner.middleware.set_custom_attribute')
     def test_load_config_with_invalid_dict(self, mock_set_custom_attribute):
-        with patch(
-                'edx_django_utils.monitoring.code_owner.utils._PATH_TO_CODE_OWNER_MAPPINGS',
-                _process_code_owner_mappings()
-        ):
-            request = RequestFactory().get('/test/')
-            self.middleware(request)
-            expected_path_module = self._REQUEST_PATH_TO_MODULE_PATH['/test/']
-            self._assert_code_owner_custom_attributes(
-                mock_set_custom_attribute, path_module=expected_path_module,
-                has_path_error=True, has_transaction_error=True
-            )
+        request = RequestFactory().get('/test/')
+        self.middleware(request)
+        expected_path_module = self._REQUEST_PATH_TO_MODULE_PATH['/test/']
+        self._assert_code_owner_custom_attributes(
+            mock_set_custom_attribute, path_module=expected_path_module,
```

Comment:
**Reviewer Note**: Don't worry about the fact that we are no longer providing errors in the custom attributes for this very unexpected case. This case will log an exception, which is tested elsewhere. Also, we would already be alerted from New Relic, providing we look for transactions with a path module and no code_owner, which we plan to do.